### PR TITLE
removed deprecated gulp-util

### PR DESCRIPTION
**Pull request by zbennett10**
This is a pull request per the recent deprecation of gulp-util.

https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
https://github.com/gulpjs/gulp-util/issues/143

Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +19,10 @@
     "chalk": "^0.5.1",
     "debug": "^2.1.0",
     "event-stream": "^3.1.7",
-    "gulp-util": "^3.0.2",
+    "fancy-log": "^1.3.2",
     "lodash.assign": "^3.0.0",
-    "mini-lr": "^0.1.8"
+    "mini-lr": "^0.1.8",
+    "vinyl": "^2.1.0"
```
Comment (demurgos) [inline package.json]: Is `vinyl` used outside of `test/index.js`? I may be missing something but i think that `vinyl` should be in the `devDependencies`.

**Comment by nullbio** [comment]
Can we please get this merged?

File: package.json (hunk 1 above)
Comment (demurgos) [reply package.json]: Ping @zbennett10

**Comment by demurgos** [comment]
@vohof 
This PR is good and fairly small. Could you merge and publish it?

Are you maybe looking for maintainers?

**Comment by esteinborn** [comment]
Looks like @vohof IS looking to transfer to another maintainer: #120

**Comment by demurgos** [comment]
@esteinborn 
Thanks for message. I replied to the issue asking for maintainers but looking at @vohof's contribution history he seems to no longer be an active Github user.

**Pull request merged**